### PR TITLE
chore: update the skaffold deployment to include mail service

### PR DESCRIPTION
**Pull request by wood-push-melon**
To resolve this [issue](https://github.com/canonical/identity-platform-admin-ui/issues/409).

mailhog

### CODE REVIEW

File: skaffold.yaml [hunk 1]
```diff
@@ -96,3 +100,8 @@ portForward:
   namespace: default
   port: 8080
   localPort: 14457
+- resourceType: service
+  resourceName: mailhog
+  namespace: default
+  port: 8025
```
Comment (shipperizer) [inline skaffold.yaml:106]: is it the UI of mailhog?

**Review by shipperizer — APPROVED** [review]

**Comment by shipperizer** [comment]
if u can, make the commit message a `chore:` or anything that doesn't trigger a release upgrade

File: skaffold.yaml (hunk 1 above)
Comment (wood-push-melon) [reply skaffold.yaml:106]: yeah, it is.

File: skaffold.yaml [hunk 2]
```diff
@@ -59,6 +59,10 @@ deploy:
         repo: https://openfga.github.io/helm-charts
         valuesFiles: ["deployments/helm/openfga/values.yaml"]
         wait: false
+      - name: mailhog
```
Comment (BarcoMasile) [inline skaffold.yaml:62]: question: why `mailhog` and not the `maildev` used in the original PR?

Comment (wood-push-melon) [reply skaffold.yaml:62]: `maildev` does not maintain the helm chart well.

**Review by BarcoMasile — APPROVED** [review]

**Pull request merged**